src/courtroom.cpp

```diff
@@ -177,6 +177,11 @@ Courtroom::Courtroom(AOApplication *p_ao_app) : QMainWindow()
   ui_ic_chat_name->setPlaceholderText(tr("Showname"));
   ui_ic_chat_name->setText(p_ao_app->get_default_showname());
   ui_ic_chat_name->setObjectName("ui_ic_chat_name");
+  
+  ui_custom_blips = new QLineEdit(this);
```

Comment:
Oh yeah also this definitely should be a dropdown. It's not ideal to just throw the player into the darkness and make them scroll through the files themselves.